src/store/item-store.js

```diff
@@ -69,6 +69,28 @@ class ItemStore extends Store {
     }
 
 }
+function getItemToEdit(itemAddress){
```

Comment:
Stores shouldn't be returning the exact item since that would mean people could edit the data outside of the store. We should be using trevin's clone method in all of the store's getter methods, but we haven't had time to do that refactor yet.

To avoid the repeated code, the existing `getItemByAddress` would just use this method(renamed this method) and then call clone on it's return value.